client/views/HistoryView.vue

```diff
@@ -1,19 +1,35 @@
 <template>
   <div id="historyView">
-    <h1>History</h1>
-    <form>
-      <p>
-        <span>From: </span>
-        <input type="date" @change="load" v-model="fromDate"/>
-        <span>To: </span>
-        <input type="date" @change="load" v-model="toDate"/>
-      </p>
-    </form>
-    <ul>
-      <li v-for="oodlet in oodlets">
-        <history-oodlet :oodlet="oodlet"></history-oodlet>
-      </li>
-    </ul>
+    <div class="header page-header">
+      <div class="row">
+        <div class="col-md-3">
+          <h1>History</h1>
+        </div>
+        <div class="form-group col-md-offset-5 col-md-2">
+          <div class="input-group">
+            <span class="input-group-addon">From: </span>
+            <input type="date" class="form-control"  @change="load" v-model="fromDate">
+          </div>
+        </div>
+        <div class="form-group col-md-2">
+          <div class="input-group">
+            <span class="input-group-addon">To: </span>
+            <input type="date" class="form-control" @change="load" v-model="toDate">
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row" id="historyOodlets">
+      <div class="col-md-12">
+        <div class="row">
+          <ul class="list">
+            <li class="col-md-2 list__item" v-for="oodlet in oodlets">
```

Comment:
No need for a new `list__item` class. It's an `li`, you might as well target it (and nest under parent `ul`'s class).